apps/vpnmanager/src/lib/data/spreadsheet.ts

```diff
@@ -3,10 +3,13 @@ import { google } from "googleapis";
 import { SheetRow } from "@/vpnmanager/types";
 import { toCamelCase } from "@/vpnmanager/utils";
 
+const spreadsheetId = process.env.NEXT_APP_GOOGLE_SHEET_ID;
```

Comment:
```suggestion
const SHEET_ID = process.env.NEXT_APP_GOOGLE_SHEET_ID;
```
There are immutable, correct?